packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -183,12 +183,12 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
             </div>
             <SendBox
               {...sendBoxProps}
-              /* @conditional-compile-remove(file-sharing) */
-              onRenderFileUploads={() => <FileUploadCards />}
               autoFocus={options?.autoFocus}
               styles={sendBoxStyles}
               /* @conditional-compile-remove(file-sharing) */
               activeFileUploads={useSelector(fileUploadsSelector).files}
+              /* @conditional-compile-remove(file-sharing) */
+              onCancelFileUpload={useFileUploadAdapter().cancelFileUpload}
```

Comment:
No need to use `useFileUploadAdapter()`
this file already has `adapter` instance in it.
Just use `adapter.cancelFileUpload`